### PR TITLE
move references to lf-edge/eve/api to lf-edge/eve-api separate repo; same for eve/libs to eve-libs

### PR DESCRIPTION
**Pull request by deitch**
This replaces #3375 which will be closed.

Now that eve-libs and eve-api are moved and up to date in their own repos, this should point to it. Most things already did point to a full path explicitly, rather than relative paths, so this (almost) completes it.

Unfortunately, it does not 100% complete it. pkg/edgeview depends both on pkg/pillar and eve/libs + eve/api, specific versions (as it should). This means that we cannot update the eve/libs and eve/api deps here until we can access the correct version of pillar. So only after this PR is in, can we update edgeview.

Also removes eve/libs and eve/api from measure-config, as they were manual replace lines, and it has no dependency on them anyways.

After this is in, we will need to update `pkg/edge-view` with a separate PR.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -3,7 +3,6 @@
 # be requested for review when someone opens a pull request.
 
 *                             @eriknordmark
-/api/                         @deitch
 /build-tools/                 @deitch
 /libs/depgraph/               @milan-zededa
```
Comment (milan-zededa) [inline .github/CODEOWNERS:7]: Do you plan to remove references to /libs/* from here and add them to CODEOWNERS of eve-libs in a separate PR?

**Review by milan-zededa — COMMENTED** [review]
There are few more references to libs in `pkg/pillar/docs` that need update - search for `/libs/` to find them.

File: .github/CODEOWNERS (hunk 1 above)
Comment (deitch) [reply .github/CODEOWNERS:7]: Yes, that was a slip-up. Fixing now.

**Comment by deitch** [comment]
I do not know what yetus is complaining about. This is the reported issue:

```
  |  -1  |       bufcompat  |   0m  0s   | Error running buf. Please check buf 
  |      |                  |            | stderr files.
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (deitch) [reply .github/CODEOWNERS]: https://github.com/lf-edge/eve-libs/pull/3

and

https://github.com/lf-edge/eve-api/pull/5

**Comment by milan-zededa** [comment]
> I do not know what yetus is complaining about. This is the reported issue:
> 
> ```
>   |  -1  |       bufcompat  |   0m  0s   | Error running buf. Please check buf 
>   |      |                  |            | stderr files.
> ```

Probably it is just confused with proto files being removed:
```
api/proto/attest/attest.proto:6:8:certs/certs.proto: does not exist
api/proto/auth/auth.proto:6:8:evecommon/evecommon.proto: does not exist
api/proto/certs/certs.proto:6:8:evecommon/evecommon.proto: does not exist
api/proto/config/acipherinfo.proto:6:8:evecommon/evecommon.proto: does not exist
api/proto/config/appconfig.proto:10:8:config/acipherinfo.proto: does not exist
api/proto/config/baseosconfig.proto:10:8:config/devcommon.proto: does not exist
api/proto/config/devcommon.proto:10:8:evecommon/devmodelcommon.proto: does not exist
api/proto/config/devmodel.proto:6:8:evecommon/devmodelcommon.proto: does not exist
api/proto/config/netconfig.proto:10:8:config/acipherinfo.proto: does not exist
api/proto/config/netinst.proto:10:8:config/devcommon.proto: does not exist
api/proto/config/storage.proto:10:8:config/devcommon.proto: does not exist
api/proto/profile/local_profile.proto:6:8:info/info.proto: does not exist
```
Seems we will have to ignore this error.

**Review by eriknordmark — COMMENTED** [review]
If we delete the api directory here, will adam/eden immediately fail to build? Or do they have a vendor directory for the API and/or locked to a particular tag of lf-edge/eve?

@giggsoff  can you comment?

**Comment by giggsoff** [comment]
> If we delete the api directory here, will adam/eden immediately fail to build? Or do they have a vendor directory for the API and/or locked to a particular tag of lf-edge/eve?
> 
> @giggsoff can you comment?

Adam/Eden pointed onto specific commits of lf-edge/eve repo (with api and libs inside), so no issues. Anyway, they are just binaries (docker images) in terms of tests in EVE-OS repo. We can adjust references in Adam/Eden eventually to be able to implement features from modified api just with go get -u.
I built and checked the version of EVE-OS from the PR with the current (unmodified) version of Eden/Adam and cannot find any problems (checked local profile logic as well as it also involved in modifications of api).

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**